lib/App/DuckPAN/Cmd/New.pm

```diff
@@ -1,22 +1,112 @@
 package App::DuckPAN::Cmd::New;
 # ABSTRACT: Take a name as input and generates a new, named Goodie or Spice instant answer skeleton
 
-# For Goodies:
-# 	- <name>.pm file is created in lib/DDG/Goodie
-#
-# For Spice:
-# 	- <name>.pm file is created in lib/DDG/Spice
-# 	- directory /share/spice/<name> is created
-# 	- <name.js> is created in /share/spice/<name>
-# 	- <name.handlebars> is created in /share/spice/<name>
+# See the template/templates.yml file in the Goodie or Spice repository for the
+# list of template-sets and files generated for them
 
 use Moo;
 with qw( App::DuckPAN::Cmd );
 
 use MooX::Options protect_argv => 0;
-use Text::Xslate qw(mark_raw);
-use Path::Tiny;
+use Try::Tiny;
 
+use App::DuckPAN::TemplateDefinitions;
+
+##########################
+# Command line arguments #
+##########################
+
+# A 'template' for the user is equivalent to a 'template-set' for the program
+option template => (
```

Comment:
a shortform, `-t` would be nice --- I'll try and compile of features that can be added later